core/src/main/java/com/amplifyframework/auth/AuthPasswordlessFlow.java

```diff
@@ -23,10 +23,10 @@ public enum AuthPasswordlessFlow {
     /**
      * Sign up and Sign in.
      */
-    SIGNUP_AND_SIGNIN,
+    SIGNUP_AND_SIGN_IN,
```

Comment:
`SIGN_UP_AND_SIGN_IN` - please do confirm with the naming conventions on Android though. I maybe wrong.

Reply:
Makes sense to me. Yeah I will update it.